ray-operator/apis/ray/v1/rayservice_types.go

```diff
@@ -13,7 +13,7 @@ type ServiceStatus string
 const (
 	WaitForServeDeploymentReady ServiceStatus = "WaitForServeDeploymentReady"
 	Running                     ServiceStatus = "Running"
-	Restarting                  ServiceStatus = "Restarting"
+	PreparingNewCluster         ServiceStatus = "PreparingNewCluster"
```

Comment:
Could this break anyone that might be watching for the "Restarting" status in RayService?

Reply:
Could we keep this as `Restarting` and use `PreparingNewCluster` for the new conditions only?